### ISSUE TITLE
Azure DNS provider -- getting issue while following sample

### ISSUE THREAD

**Issue by mithrandyr** [rmbolger/Posh-ACME#80]
I'm using credentials instead of token or MIS.  Following the information here: https://github.com/rmbolger/Posh-ACME/blob/master/Posh-ACME/DnsPlugins/Azure-Readme.md

while running this 
``` powershell
$azParams = @{
  AZSubscriptionId='xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx';
  AZTenantId='xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx';
  AZAppCred=(Get-Credential)
}

# issue a cert
New-PACertificate test.example.com -DnsPlugin Azure -PluginArgs $azParams
```
I'm getting the following error/output:

```
VERBOSE: Using directory https://acme-staging-v02.api.letsencrypt.org/directory
VERBOSE: POST https://acme-staging-v02.api.letsencrypt.org/acme/acct/6294604 with -1-byte payload
VERBOSE: received 330-byte response of content type application/json
VERBOSE: Using account 6294604
VERBOSE: Using existing order for *.xxxx.xxxx.com with status pending
VERBOSE: Publishing DNS challenge for *.xxxx.xxxx.com
Add-DnsTxtAzure : A parameter cannot be found that matches parameter name 'AZSubscriptionId'.
At C:\Program Files\WindowsPowerShell\Modules\posh-acme\2.7.1\Public\Publish-DNSChallenge.ps1:44 char:5
+     &$addCommand $recordName $txtValue @PluginArgs
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Add-DnsTxtAzure], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Add-DnsTxtAzure
```

I don't understand why I'm getting this error 
```Add-DnsTxtAzure : A parameter cannot be found that matches parameter name 'AZSubscriptionId'.```
-- following the guide explicitly, running this from windows 10 (1803) -- latest version of Posh-ACME(2.7.1)

**Comment by mithrandyr** [comment]
Full error details
```
PSMessageDetails      :
Exception             : System.Management.Automation.ParameterBindingException: A parameter cannot be found that
                        matches parameter name 'AZSubscriptionId'.
                           at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext
                        funcContext, Exception exception)
                           at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame
                        frame)
                           at
                        System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame
                        frame)
                           at
                        System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame
                        frame)
                           at System.Management.Automation.Interpreter.Interpreter.Run(InterpretedFrame frame)
                           at System.Management.Automation.Interpreter.LightLambda.RunVoid1[T0](T0 arg0)
                           at System.Management.Automation.PSScriptCmdlet.RunClause(Action`1 clause, Object
                        dollarUnderbar, Object inputToProcess)
                           at System.Management.Automation.PSScriptCmdlet.DoEndProcessing()
                           at System.Management.Automation.CommandProcessorBase.Complete()
TargetObject          :
CategoryInfo          : InvalidArgument: (:) [Connect-AZTenant], ParameterBindingException
FullyQualifiedErrorId : NamedParameterNotFound,Connect-AZTenant
ErrorDetails          :
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at Add-DnsTxtAzure, C:\Program
                        Files\WindowsPowerShell\Modules\Posh-ACME\2.7.1\DnsPlugins\Azure.ps1: line 22
                        at Publish-DnsChallenge, C:\Program
                        Files\WindowsPowerShell\Modules\Posh-ACME\2.7.1\Public\Publish-DNSChallenge.ps1: line 44
                        at Submit-ChallengeValidation, C:\Program
                        Files\WindowsPowerShell\Modules\Posh-ACME\2.7.1\Public\Submit-ChallengeValidation.ps1: line 122
                        at New-PACertificate, C:\Program
                        Files\WindowsPowerShell\Modules\Posh-ACME\2.7.1\Public\New-PACertificate.ps1: line 159
                        at <ScriptBlock>, <No file>: line 1
PipelineIterationInfo : {}
```

**Comment by rmbolger** [comment]
Bleh, thanks for the heads up on this. Some of the recent changes to this plugin introduced a bug that wasn't apparent when only testing with `Publish-DnsChallenge`. I've committed a fix that should go out later today as a new release.

**Comment by rmbolger** [comment]
This is now fixed in 2.8.0
